canonical-web-and-design/vanilla-framework
# Add new social RSS icon

## Done
- Start of updating social icons to use the official brand assets (other icons will follow in new PR)
- Added new social RSS icon
- No hover state

## QA
- Pull code
- Run `./run serve --watch`
- Open local http://0.0.0.0:8101/patterns/icons/#social or open demo https://vanilla-framework-canonical-web-and-design-pr-2639.run.demo.haus/examples/patterns/icons/icons-social/  
- You should see a new social icon and inside a demo example of docs, as we can't serve this push due to docs being on a previous release of our framework

## Details
Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2560

## Visual
<img width="1405" alt="Screenshot 2019-11-12 at 15 59 47" src="https://user-images.githubusercontent.com/17748020/68687966-ea5caf80-0565-11ea-8074-62467a7ba074.png">
